docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
 Directed Acyclic Graphs (Causal Diagrams)
 ------------------------------------------
 
-.. todo::
+.. important::
+
+  The figures represented in this section represent directed acyclic graphs/causal diagrams. These figures are distinct from compartamental model diagrams (such as SIR models). For our purposes we will represent directed acyclic graphs as *circles* connected by arrows and compartamental model diagrams as *squares* connected by arrows.
+
+  Additionally, for the purposes of directed acyclic graphs, variables will be labeled with capital letters (A, B, C, etc.) and arrows will be labeled with lower case letters (a, b, c, etc.).
+
+Introduction to Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++
+
+Directed acyclic graphs (DAGs), also termed causal diagrams, are composed of variables that are linked together by directional arrows. Critically, the solid arrows in DAGs represent a causal relationship between an exposure (start of the arrow) and outcome (node of the arrow), representing the *directional* aspect of DAGs. Notably, a directional arrow in a DAG does not offer any information about whether the exposure causes an increased or decreased probability of the outcome, only that there is some causal realtionship bewteen the exposure and outcome. 
+
+The *acyclic* nature of DAGs is that there cannot be feedback loops between any variables in the graph. In this way, the arrows in DAGs can be interpreted as the passage of time so that the exposure (cause) must always occur temporally *before* the outcome (effect). See the diagram below for an example of an inappropriate and appropriate DAG with regard to temporality.
+
+.. image:: acyclic_dags.svg
+
+Applications, Assumptions, and Limitations of Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+DAGs are used in epidemiology to represent a network of relationships between several variables, which in turn helps to guide the selection of variables that will need to be measured or considered in a given study/project. Further, the exact relationship between variables in a DAG will dictate *how* to treat or consider a specific variable in that study/project by helping to indentify potential confounding, intermediate, or mediating variables between a given exposure and outcome of interest (see the `Causal Relationships`_ section for definitions).
+
+However, DAGs are limited in that they require *prior causal knowledge* about the variables that may be causally related to a given exposure and outcome of interest. Therefore, there may be uncertain, unmeasured, or unknown variables that cannot be represented in a given DAG. Further, DAGs do not allow for easy representation of direction or magnitude of a causal relationship or for effect modification.
```

Comment:
Beatrix had indicated that the phrase "direction or magnitude of a causal relationship" could be confusing because "direction" could be interpreted as the direction of the causal arrow, rather than whether the arrow implies an increased or decreased probability of the outcome. I think it helps that you previously mentioned this "increased or decreased probability," but perhaps there's a different way to word this to make that even more clear?